extensions/wikia/CommunityPage/styles/entrypoint/EntryPoint.scss

```diff
@@ -36,7 +36,7 @@ $logo-url: $logo-url-light;
 		left: 50%;
 		position: absolute;
 		top: 0;
-		transform: translate(-50%, -50%);
+		transform: translate(-50%, -40%);
```

Comment:
Discussed with @lukaszkonieczny: Leaving it like this for now. We may change transform translate at some later point based on further discussion.